src/screens/AuthScreen/auth0-credentials.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
This isn't on the node side you can use es6 export syntax. Babel is already being used.